### ISSUE TITLE
Cachet says some services are having issues even though all components are in "Operational" state

### ISSUE THREAD

**Issue by lelutin** [cachethq/Cachet#790]
Throughout normal usage, we ended up with some kind of weird situation. The status page says there is some problem somewhere but all components are set to "Operational" status.

See: https://status.koumbit.net/

I've looked around the dashboard and couldn't see any indication of why cachet would be saying there is an issue.

I've also tried changing status for all components, and then changing them all back to "Operational" but the situation didn't resolve itself.

**Comment by GrahamCampbell** [comment]
That's because you need to post a new resolved incident. We're showing failing because the last incident posted was unresolved.

**Comment by lelutin** [comment]
oh .. indeed. I created a new incident and made it use the "Operational" state and now status is OK.

before creating this I also tried to edit the latest incident in order to mark its state as "Operational" but it wouldn't save that.

also, that's rather a weird interface perk.. all services were shown as being OK but there was still the problematic status and nothing in the interface would give a clue as to why that warning was still there and how to resolve it. there's probably some work to do to expose this in the interface.
